solidity/tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
+  "files": ["./hardhat.config.ts"],
```

Comment:
We have `"node_modules/@keep-network/hardhat-local-networks-config/src/type-extensions.d.ts"` in `/yearn` dir. Probably we'd need to add it here as well.

Reply:
Good catch! Addressed in https://github.com/keep-network/tbtc-v2/pull/17/commits/36bbaac0c6e54181b6f414ea99927e288ace7a97